src/riak_kv_eleveldb_backend.erl

```diff
@@ -47,13 +47,12 @@
          is_empty/1,
          status/1,
          callback/3]).
-
 -export([data_size/1]).
 
--compile({inline, [
-                   to_object_key/2, from_object_key/1,
-                   to_index_key/4, from_index_key/1
-                  ]}).
+% -compile({inline, [
```

Comment:
For the inlining, looks like it was added around 2I time - I doubt we have
any record of benchmarks.

On Thu, Feb 18, 2016 at 4:50 AM Andy Till <notifications@github.com> wrote:

> In src/riak_kv_eleveldb_backend.erl
> <https://github.com/basho/riak_kv/pull/1349#discussion_r53304129>:
>
> >  -export([data_size/1]).
> >
> > --compile({inline, [
> > -                   to_object_key/2, from_object_key/1,
> > -                   to_index_key/4, from_index_key/1
> > -                  ]}).
> > +% -compile({inline, [
>
> Just a development change so I can trace these functions. Does it really
> improve performance?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/basho/riak_kv/pull/1349/files#r53304129>.
>